docs/howtos/http-requests.md

```diff
@@ -0,0 +1,170 @@
+# How to make HTTP requests using pyscript, in pure Python
+
+[Pyodide](https://pyodide.org), the runtime that underlies PyScript, does not have the `requests` module
+(or other similar modules) available by default, which are traditionally used to make HTTP requests in Python.
+However, it is possible to make HTTP requests in Pyodide using the modern javascript `fetch` API
+([docs](https://developer.mozilla.org/en-US/docs/Web/API/fetch)). This example shows how to make common HTTP request
+(GET, POST, PUT, DELETE) to an API, using only python code! We will use asynchronous functions with
+async/await syntax, as concurrent code is preferred for HTTP requests.
+
+The purpose of this guide is not to teach the basics of HTTP requests, but to show how to make them
+from `pyscript` using python, since currently, the common tools such as `requests` and `httpx` are not available.
+
+## Fetch
+
+The `fetch` API is a modern way to make HTTP requests. It is available in all modern browsers, and in Pyodide.
+
+Although there are two ways to use `fetch`, 1) using javacript from pyscript, and 2) using Pyodide's python wrapper,
```

Comment:
`javacript` --> `javascript`

Reply:
Good call, will do the latter.

Reply:
Thanks for the catch and suggestion, I've made the changes. Looks better.